docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Gap
+
+    gap: {
+      options: ['none', 'condensed', 'normal', 'spacious', 'custom'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: `Sets the spacing gap between items. All sizes are rendered in \`rem\` units.
+- \`none\`: 0
+- \`condensed\`: \`var(--primer-stack-gap-condensed, 8px)\`,
+- \`normal\`: \`var(--primer-stack-gap-normal, 16px)\` (default)
+- \`spacious\`: \`var(--primer-stack-gap-spacious, 24px)\` (on regular viewports, otherwise it appears as \`normal\` on narrow viewports)
+- \`custom\`: set a custom size. When using with a framework such as ViewComponent or React, a custom value can be passed directly to the property.
+ `,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'normal',
+        }
+      }
+    },
+    narrow_gap: {
+      options: ['inherit', 'none', 'condensed', 'normal', 'custom'],
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'Override `gap` on narrow viewports',
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+    gap_custom: {
+      control: {
+        type: 'text'
+      },
+      description: 'A custom value to `gap`. Refer to [Primer Primitives](https://primer.style/primitives/spacing) for other spacing tokens. Example: `var(--base-size-12, 12px)`.',
+      table: {
+        category: 'Properties',
+      },
+    },
+    narrow_gap_custom: {
+      control: {
+        type: 'text'
+      },
+      description: 'Override a custom value for `gap` for narrow viewports',
+      table: {
+        category: 'Narrow viewport properties'
+      },
+    },
+
+    align: {
+      options: ['stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: `Sets the alignment between items in the cross-axis of the specified direction. For example:
+- If \`direction\` is set to \`block\` (stacks vertically), it controls the horizontal alignment (left, center, right).
+- If \`direction\` is set to \`inline\` (stacks horizontally), it controls the vertical alignment (top, center, bottom).
+
+This property behavior is equivalent to the \`align-items\` Flexbox property.`,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'stretch',
+        }
+      }
+    },
+    narrow_align: {
+      options: ['inherit', 'stretch', 'start', 'center', 'end', 'baseline'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit',
+        }
+      },
+    },
+
+    // Align wrap
+
+    alignWrap: {
+      options: ['start', 'center', 'end', 'distribute', 'distributeEvenly'],
```

Comment:
Non-blocking nitpick: I actually think 'spaceBetween` and `spaceEvenly` are easier to understand, but that could just be me.